app/support/payoff_manager.rb

```diff
@@ -0,0 +1,103 @@
+## Loan Manager to determine funding channel status
+## Determines using current month history and construction activity
+require 'byebug'
+module PayoffManager
+  include Days360
+  DISCHARGE_FEE = 250
+
+  def create_payoff(date_string)
+    date = Date.parse(date_string)
+    @payoff = payoffs.build(
+      payoff_date: date
+    )
+    create_loan_amount_line_item
+    create_unused_rehab_budget_line_item
+    create_interest_line_items(date)
+    create_closing_fee_line_items
+    @payoff.save!
```

Comment:
do we have to save this here?  Lets discuss.

Reply:
how many saves might be in all those methods above?  Always think about how many times you are going to the database as it is costly.